queries.sql

```diff
@@ -0,0 +1,9 @@
+SET SQL_SAFE_UPDATES = 0;
+
+UPDATE teams 
```

Comment:
No need for the subquery here since you are looking to change from WAS to WSH.

`UPDATE teams SET teamAbbreviation = 'WAS" WHERE teamAbbreviation = 'WSH';`